pkgs/by-name/qr/qrtool/package.nix

```diff
@@ -4,20 +4,21 @@
 , rustPlatform
 , asciidoctor
 , installShellFiles
+, nix-update-script
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "qrtool";
-  version = "0.8.5";
+  version = "0.10.0";
 
   src = fetchFromGitHub {
     owner = "sorairolake";
     repo = "qrtool";
     rev = "v${version}";
-    sha256 = "sha256-jrvNZGO1VIDo6Mz3NKda1C7qZUtF9T00CAFK8yoGWjc=";
+    sha256 = "sha256-PXjggeDofrrS32XdqnOgAMlAiw+/7eMQpELw8EmiwNI=";
```

Comment:
```suggestion
    hash = "sha256-PXjggeDofrrS32XdqnOgAMlAiw+/7eMQpELw8EmiwNI=";
```

Reply:
I think I'll leave them. There's no guidance one way or the other that I can find in the docs and this way works just fine.

Reply:
https://nixos.org/manual/nixpkgs/stable/#compiling-rust-applications-with-cargo

```
buildRustPackage requires either the cargoSha256 or the cargoHash attribute which is computed over all crate sources of this package. cargoHash256 is used for traditional Nix SHA-256 hashes, such as the one in the example above. cargoHash should instead be used for [SRI](https://www.w3.org/TR/SRI/) hashes. For example:
```

same logic applies for `sha256`